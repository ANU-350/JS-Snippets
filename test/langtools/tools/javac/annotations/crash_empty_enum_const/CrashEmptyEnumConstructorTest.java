/*
 * @test /nodynamiccopyright/
 * @bug 8305971
 * @summary NPE in JavacProcessingEnvironment for missing enum constructor body
 * @compile SimpleProcessor.java
 * @compile/fail/ref=CrashEmptyEnumConstructorTest.out -processor SimpleProcessor -XDrawDiagnostics CrashEmptyEnumConstructorTest.java
 */

enum CrashEmptyEnumConstructorTest {
    ONE("");
    CrashEmptyEnumConstructorTest(String one);
    final String one;
}
