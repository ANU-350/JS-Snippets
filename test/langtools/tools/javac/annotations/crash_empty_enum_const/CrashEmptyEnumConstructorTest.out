CrashEmptyEnumConstructorTest.java:11:5: compiler.err.missing.meth.body.or.decl.abstract
1 error
